### PR TITLE
Rename rte-operator configuration to numaresources-operator

### PR DESCRIPTION
**Pull request by Tal-or**
Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
thanks for creating the PR so fast! It seems to me a few `git mv`s are missing:
```
./ci-operator/config/openshift-kni/rte-operator/openshift-kni-rte-operator-main.yaml		OK
./ci-operator/config/openshift-kni/rte-operator/OWNERS						OK
./ci-operator/jobs/openshift-kni/rte-operator/openshift-kni-rte-operator-main-postsubmits.yaml	OK
./ci-operator/jobs/openshift-kni/rte-operator/openshift-kni-rte-operator-main-presubmits.yaml	OK
./ci-operator/jobs/openshift-kni/rte-operator/OWNERS						MISSING
./core-services/prow/02_config/openshift-kni/rte-operator/OWNERS				MISSING
./core-services/prow/02_config/openshift-kni/rte-operator/_pluginconfig.yaml			MISSING
./core-services/prow/02_config/openshift-kni/rte-operator/_prowconfig.yaml			MISSING
```

**Comment by Tal-or** [comment]
> thanks for creating the PR so fast! It seems to me a few `git mv`s are missing:
> 
> ```
> ./ci-operator/config/openshift-kni/rte-operator/openshift-kni-rte-operator-main.yaml		OK
> ./ci-operator/config/openshift-kni/rte-operator/OWNERS						OK
> ./ci-operator/jobs/openshift-kni/rte-operator/openshift-kni-rte-operator-main-postsubmits.yaml	OK
> ./ci-operator/jobs/openshift-kni/rte-operator/openshift-kni-rte-operator-main-presubmits.yaml	OK
> ./ci-operator/jobs/openshift-kni/rte-operator/OWNERS						MISSING
> ./core-services/prow/02_config/openshift-kni/rte-operator/OWNERS				MISSING
> ./core-services/prow/02_config/openshift-kni/rte-operator/_pluginconfig.yaml			MISSING
> ./core-services/prow/02_config/openshift-kni/rte-operator/_prowconfig.yaml			MISSING
> ```

```make jobs``` should done the work. weird

**Comment by fromanirh** [comment]
thanks @Tal-or for taking care!

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ blunderbuss:
   max_request_count: 2
   request_count: 2
   use_status_availability: true
+branch_cleaner: {}
```
Comment (fromanirh) [inline core-services/prow/02_config/_plugins.yaml:5]: this is why you need extra approvals @Tal-or 
Do we really need this change?

Comment (Tal-or) [reply core-services/prow/02_config/_plugins.yaml:5]: It was automatically generated

Comment (Tal-or) [reply core-services/prow/02_config/_plugins.yaml]: I'll try remove it, let's if it'll pass

**Pull request merged**